src/Checkbox.jsx

```diff
@@ -18,7 +18,7 @@ export default class Checkbox extends React.Component {
     onBlur: PropTypes.func,
     onChange: PropTypes.func,
     onClick: PropTypes.func,
-    tabIndex: PropTypes.string,
+    tabIndex: PropTypes.number,
```

Comment:
For compatibility, it's better to keep `PropTypes.string`.

`PropTypes.oneOfType([PropTypes.string, PropTypes.number])`.